examples/pipelineruns/limitrange.yaml

```diff
@@ -1,7 +1,13 @@
 apiVersion: v1
+kind: Namespace
+metadata:
+  name: pipelinerun-limitrange-ns
+---
+apiVersion: v1
 kind: LimitRange
 metadata:
   name: limit-mem-cpu-per-container
+  namespace: pipelinerun-limitrange-ns
```

Comment:
Our "check" won't check this though, as "did it run sucessfully", as, it only look into the current namespace (`default` or one created by the CI :thinking:).

Reply:
By chance, do you know if other namespaces could be added to check?

Reply:
> By chance, do you know if other namespaces could be added to check?

Yep they could :angel:

Reply:
Let me look into [e2e-common.sh](https://github.com/tektoncd/pipeline/blob/master/test/e2e-common.sh) and see what I can do to try and add the results.